internal/lib/lib.go

```diff
@@ -725,10 +732,19 @@ func GetMarkersChecksum(markers utils.AviObjectMarkers) uint32 {
 	var cksum uint32
 	var combinedString string
 	numMarkerFields := vals.NumField()
+	typeOfVals := vals.Type()
 	markersStr := make([]string, numMarkerFields)
 	for i := 0; i < numMarkerFields; i++ {
 		if vals.Field(i).Interface() != "" {
-			value := vals.Field(i).Interface().(string)
+			field := typeOfVals.Field(i).Name
+			var value string
+			if field == "Path" || field == "IngressName" {
```

Comment:
Unable to understand this code block. But I am assuming something is expected to have pathsArr for either the `Path` or the `IngressName` field.

Reply:
Right. With this PR, for HTTP PS markers , ingressName key and path key will have multiple values. To take into consideration all that values, above code logic is used.